basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,264 @@
 const helpers = require('./project-1');
 
 // start testing!
+//#1
+it('multiply by 10', () => {
+    const numTimesTen = helpers.multiplyByTen;
+    const timesTwo = numTimesTen(2);
+    const timesFive = numTimesTen(5);
+    const timesZero = numTimesTen(0);
+    const timesNaN = numTimesTen(undefined);
+    const timesStr = numTimesTen('str');
+
+    expect(timesTwo).toEqual(20);
```

Comment:
I like this, resolving the outcome before the test conditional could save us async composition issues.